.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[*]
+indent_style = space
+trim_trailing_whitespace = true
+indent_size = 2
```

Comment:
Why add this back? @code-asher

Reply:
This is what I use to set default indentation settings. Is there a more reliable/supported method? I personally don't try to auto-detect indentation because I don't see the advantage over an explicit setting.

Reply:
Prettier takes care of this, never have to think about it.

Reply:
Sure, but I'd love to have the correct indentation as I type.

Reply:
These are just the normal javascript/typescript settings, isn't this your default?

editconfig is only really useful when you're going against your community's norms.

Reply:
The historical standard is four spaces as far as I'm aware. If you open code-server in VS Code or Intellij Idea and create a new Typescript file for example, both will use four spaces. For existing files they attempt to look at the existing indentation so usually that works fine as long as the file has some indentation, but since it covers fewer use cases than the config I prefer not to rely on it.

Reply:
Kk, will add back.

Reply:
Although even if I did have my defaults configured against the norms (for example if I preferred tabs and had that set as my default), it would still be useful to have the correct indentation when I'm working on this project.